test/unit/uk/gov/hmrc/bindingtariffclassification/scheduler/DaysElapsedJobTest.scala

```diff
@@ -48,65 +46,69 @@ class DaysElapsedJobTest extends UnitSpec with MockitoSugar with BeforeAndAfterE
   "Scheduled Job" should {
 
     "Configure 'Name'" in {
-      new DaysElapsedJob(appConfig, caseService, bankHolidaysConnector).name shouldBe "DaysElapsed"
+      createDaysElapsedJob.name shouldBe "DaysElapsed"
     }
 
     "Configure 'firstRunTime'" in {
       val runTime = LocalTime.of(14, 0)
-      given(appConfig.daysElapsed).willReturn(JobConfig(runTime, FiniteDuration(1, DAYS)))
+      given(appConfig.daysElapsed).willReturn(JobConfig(runTime, 1.day))
 
-      new DaysElapsedJob(appConfig, caseService, bankHolidaysConnector).firstRunTime shouldBe runTime
+      createDaysElapsedJob.firstRunTime shouldBe runTime
     }
 
     "Configure 'interval'" in {
-      given(appConfig.daysElapsed).willReturn(JobConfig(LocalTime.MIDNIGHT, FiniteDuration(1, DAYS)))
+      given(appConfig.daysElapsed).willReturn(JobConfig(LocalTime.MIDNIGHT, 1.day))
 
-      new DaysElapsedJob(appConfig, caseService, bankHolidaysConnector).interval shouldBe FiniteDuration(1, DAYS)
+      createDaysElapsedJob.interval shouldBe 1.day
     }
 
     "Execute" in {
       givenItIsNotABankHoliday()
       givenTheDateIsFixedAt("2018-12-25T12:00:00")
-      given(appConfig.daysElapsed).willReturn(JobConfig(LocalTime.MIDNIGHT, FiniteDuration(1, DAYS)))
-      given(caseService.incrementDaysElapsed(anyDouble())).willReturn(Future.successful(2))
+      given(appConfig.daysElapsed).willReturn(JobConfig(LocalTime.MIDNIGHT, 1.day))
+      given(caseService.incrementDaysElapsed(1)).willReturn(Future.successful(2))
 
-      await(new DaysElapsedJob(appConfig, caseService, bankHolidaysConnector).execute())
+      await(createDaysElapsedJob.execute())
 
       verify(caseService).incrementDaysElapsed(1)
     }
 
     "Execute with decimal interval" in {
       givenItIsNotABankHoliday()
       givenTheDateIsFixedAt("2018-12-25T12:00:00")
-      given(appConfig.daysElapsed).willReturn(JobConfig(LocalTime.MIDNIGHT, FiniteDuration(12, HOURS)))
-      given(caseService.incrementDaysElapsed(anyDouble())).willReturn(Future.successful(2))
+      given(appConfig.daysElapsed).willReturn(JobConfig(LocalTime.MIDNIGHT, 12.hours))
+      given(caseService.incrementDaysElapsed(0.5)).willReturn(Future.successful(2))
```

Comment:
We verify this below. I prefer leaving anyDouble() so the test failure is from the verify, rather than a NullPointer